control/controlclient/direct.go

```diff
@@ -1217,9 +1223,13 @@ func (c *Direct) answerPing(pr *tailcfg.PingRequest) {
 		c.logf("invalid PingRequest with no URL")
 		return
 	}
-	if pr.Types == "" {
+	switch pr.Types {
+	case "":
 		answerHeadPing(c.logf, httpc, pr)
 		return
+	case "c2n":
```

Comment:
return an error if `c2n` and `c.noiseConfigured == false`?

Reply:
I need this for tests. There's a TODO from you in our integration tests to make noise work with testcontrol

Reply:
I guess I should go fix that 😄

Reply:
can you add a env override for now?